src/components/SingleArticleView.jsx

```diff
@@ -1,30 +1,60 @@
-import { useParams } from "react-router-dom";
+import { useParams, useNavigate } from "react-router-dom";
 import { useEffect, useState } from "react";
-import { getArticle } from "../app";
+import { getArticle, getComments } from "../app";
 import Loading from "./Loading";
 import Article from "./Article";
+import CommentList from "./CommentList";
 
 const SingleArticleView = () => {
   const { article_id } = useParams();
   const [article, setArticle] = useState({});
+  const [comments, setComments] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
 
+  const navigate = useNavigate();
+
   useEffect(() => {
     setIsLoading(true);
     getArticle(article_id).then((fetchedArticle) => {
       setArticle(fetchedArticle);
       setIsLoading(false);
     });
+    getComments(article_id).then((fetchedComments) => {
+      setComments(fetchedComments);
+    });
```

Comment:
This component does really do anything with comments other than to pass it down through props so should it be fetched in this component?